mod/bookmarks/views/default/resources/bookmarks/edit.php

```diff
@@ -6,7 +6,7 @@
  */
 
 $bookmark_guid = elgg_extract('guid', $vars);
-elgg_entity_gatekeeper($bookmark_guid, 'object', 'bookmarks');
+elgg_entity_gatekeeper($bookmark_guid, 'object', 'bookmarks', true);
 
 $bookmark = get_entity($bookmark_guid);
 
```

Comment:
i believe the canEdit check can be removed now we have the check in the gatekeeper